lib/simplecov/result.rb

```diff
@@ -85,31 +81,7 @@ def self.from_hash(hash)
 
   private
 
-    # We changed the format of the raw result data in simplecov, as people are likely
-    # to have "old" resultsets lying around (but not too old so that they're still
-    # considered we can adapt them).
-    # See https://github.com/simplecov-ruby/simplecov/pull/824#issuecomment-576049747
-    def adapt_result(result)
-      if pre_simplecov_0_18_result?(result)
-        adapt_pre_simplecov_0_18_result(result)
-      else
-        result
-      end
-    end
-
-    # pre 0.18 coverage data pointed from file directly to an array of line coverage
-    def pre_simplecov_0_18_result?(result)
-      _key, data = result.first
-
-      data.is_a?(Array)
-    end
-
-    def adapt_pre_simplecov_0_18_result(result)
-      result.transform_values do |line_coverage_data|
-        {"lines" => line_coverage_data}
-      end
-    end
-
+    # This is dropping data of files that can not be found... why though?
```

Comment:
seriously... why? Can't we just let it in there and it'll be dropped on result generation anyhow as we can't display it? Probaby have to do some git digging on this to find the reason.

Reply:
It has existed basically forever: https://github.com/simplecov-ruby/simplecov/commit/4f37de8931b60e9731b100cd63cd0badca4ce5e3